deploy/gk-deploy

```diff
@@ -499,3 +501,52 @@ else
 fi
 
 output "Ready to create and provide GlusterFS volumes."
+
+heketi-cli -s http://${heketi_service} setup-openshift-heketi-storage
```

Comment:
All the commands are passed to eval_output to log the output and also to handle failures. This has to be adapted here.